sql/dql/basic-queries/order-by-clause.md

```diff
@@ -41,117 +41,104 @@ aspects:
 ---
 ## Content
 
-`ORDER BY` clause enables users to order data in ascending or descending order on one or more columns. It comes after `GROUP BY` clause.
+The `ORDER BY` clause enables users to order data in ascending or descending order on one or more columns. This is also more commonly known as **sorting** data.
+
+Consider the following `employees` table:
+
+| id  | first_name | last_name | email                   |
+| --- | ---------- | --------- | ----------------------- |
+| 1   | Karlene    | Pantone   | kpantone0@topsy.com     |
+| 2   | Donnamarie | Beeho     | dbeeho1@state.tx.us     |
+| 3   | Tony       | Isworth   | tisworth2@bloglovin.com |
+| 4   | Esta       | Warrier   | ewarrier3@berkeley.edu  |
+| 5   | Immanuel   | Kippen    | ikippen4@biglobe.ne.jp  |
+| 6   | Regina     | Pantone   | regone@stratos.org      |
+
+Right now, the table is sorted in ascending order by the `id` column, but what if you need the results of your query to be sorted in ascending order by the `last_name` column? To achieve this you would use the following syntax: 
 
-Considering the following example:
 ```sql
-SELECT COUNT(name), language_id
-FROM move_name
-GROUP BY language_id;
+SELECT last_name, first_name
+FROM employees
+ORDER BY last_name;
 ```
 
-*Result:*
-```
-count | language_id
-======+============
-  639 |           6
-  639 |           5
-  639 |           8
-  639 |           1
-  639 |           9
-  617 |           3
-  639 |           7
-(7 rows)
-```
+*Resulting set:*
+
+| last_name | first_name |
+| --------- | ---------- |
+| Beeho     | Donnamarie |
+| Isworth   | Tony       |
+| Kippen    | Immanuel   |
+| Pantone   | Karlene    |
+| Pantone   | Regina     |
+| Warrier   | Esta       |
+
+Note that the ascending order is set as default. If we want to display the result in a descending order we need to put `DESC` after specifying the columns.
```

Comment:
I'd introduce this concept slightly more declarative:

"The sorting behavior of `GROUP BY` can be explicitly changed with the `ASC` (ascending, from small to big) or `DESC` (descending, from big to small) keywords. ...

Reply:
changed paragraph to:

The sorting behavior of `ORDER BY` can be explicitly changed by placing the `ASC` (ascending) or `DESC` (descending) keywords after the column name: `ORDER BY column_name ASC/DESC`.